package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "betterself",
+  "version": "0.0.1",
+  "description": "No Description",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jeffshek/betterself.git"
+  },
+  "author": "Jeffrey Shek",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/jeffshek/betterself/issues"
+  },
+  "homepage": "https://github.com/jeffshek/betterself#readme",
+  "dependencies": {
+    "babel": "^6.5.2",
+    "babel-cli": "^6.22.2",
+    "babel-core": "^6.23.1",
+    "babel-loader": "^6.3.2",
+    "babel-preset-es2015": "^6.22.0",
+    "babel-preset-react": "^6.23.0",
+    "jquery": "^3.1.1",
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "webpack": "^2.2.1",
+    "webpack-bundle-tracker": "^0.2.0"
+  },
+  "devDependencies": {
```

Comment:
Just remove devDependencies for now until you know what's going to be necessary / not